### PR TITLE
Relax the tx max byte size limit

### PR DESCRIPTION
**Pull request by ramtinms**
We need to relax this until we have a better way to encode smart contract deployment transactions.

### CODE REVIEW

**Review by Kay-Zee — APPROVED** [review]
Tried with https://github.com/onflow/flow-go-sdk/pull/111 and deploying the staking contract, and still got: 

`invalid transaction: transaction byte size (88643) exceeds the maximum byte size allowed for a transaction (64000)`

which is more like 90kb. I think for now, `1.5mb` is the best bet, since that fix will not be going out into the wild for a while either, and this is blocking some community members on testnet. We'll reduce this, probably to something closer to 256kb once that change for the go sdk is released

**Comment by zhangchiqing** [comment]
I think we also need this change https://github.com/onflow/flow-go/pull/149

**Comment by ramtinms** [comment]
I applied the changes you asked @zhangchiqing

File: module/builder/collection/config.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func DefaultConfig() Config {
 		ExpiryBuffer:            15,                              // 15 blocks for collections to be included
 		MaxPayerTransactionRate: 0,                               // no rate limiting
 		UnlimitedPayers:         make(map[flow.Address]struct{}), // no unlimited payers
-		MaxCollectionByteSize:   uint64(1000000),                 // ~1MB
+		MaxCollectionByteSize:   uint64(1750000),                 // ~1.75MB
```
Comment (zhangchiqing) [inline module/builder/collection/config.go:45]:
```suggestion
		MaxCollectionByteSize:   uint64(1750000),                 // ~1.75MB. This is slightly higher than the limit on single tx size, which is 1.5MB
```

**Review by zhangchiqing — APPROVED** [review]

**Pull request merged**